fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1774,6 +1905,17 @@ def _pre_load_state_dict_hook(
     replace_by_prefix_(state_dict, prefix, prefix + "_fsdp_wrapped_module.")
 
 
+def _clean_path(path: str):
+    return ".".join([split for split in path.split(".") if split not in {"_fsdp_wrapped_module", "_fpw_module"}])
+
+
+def _numel_from_size(size: torch.Size):
+    numel = 1
+    for dim in size:
+        numel *= dim
+    return numel
```

Comment:
```suggestion
    return math.prod(size)
```

Reply:
Thank you Min :) I did not know about this one !